### PR TITLE
[Outlook] (IRM) Supported now in preview

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[68ab3a9](https://github.com/OfficeDev/office-js-docs-pr/commits/68ab3a91024146b0b9c1e17f12936a1afc8d28c0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2049) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2049) |
[docs/outlook/troubleshoot-outlook-add-in-activation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/troubleshoot-outlook-add-in-activation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/troubleshoot-outlook-add-in-activation?branch=pr-en-us-2049) |
[docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/use-regular-expressions-to-show-an-outlook-add-in?branch=pr-en-us-2049) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202007281753507466-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=D33OwnV3%2BjiXzqSyrFDNDExI2ycUWWNoaclRPE5xpTI%3D&st=2020-07-28T17%3A50%3A41Z&se=2020-08-28T17%3A55%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/outlook/outlook-add-ins-overview.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Outlook add-ins are available on messages or appointments while composing or rea
 
 - Protected by Information Rights Management (IRM) or encrypted in other ways for protection. A digitally signed message is an example since digital signing relies on one of these mechanisms.
 
+  > [!IMPORTANT]
+  > Starting with Outlook build 13120.1000 on Windows, add-ins now activate on items protected by IRM. This feature is currently [in preview](../reference/requirement-sets/outlook-api-requirement-sets.md#using-preview-apis).
```
Comment (emtencza) [inline docs/outlook/outlook-add-ins-overview.md:49]: Add-ins will activate on IRM items only if the Usage Right to Allow Programmatic Access is enabled (OBJMODEL defined here): https://docs.microsoft.com/en-us/azure/information-protection/configure-usage-rights#usage-rights-and-descriptions

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8b4efa2](https://github.com/OfficeDev/office-js-docs-pr/commits/8b4efa29e6c801f587ef8ed9bb45e47ffbd8b0f3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2049) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2049) |
[docs/outlook/troubleshoot-outlook-add-in-activation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/troubleshoot-outlook-add-in-activation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/troubleshoot-outlook-add-in-activation?branch=pr-en-us-2049) |
[docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/use-regular-expressions-to-show-an-outlook-add-in?branch=pr-en-us-2049) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008031846458524-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=et0gVa0lptRd13udkRPAWsSXLruXvhynMpUBtcHkPSI%3D&st=2020-08-03T18%3A43%3A42Z&se=2020-09-03T18%3A48%3A42Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1b8d2c2](https://github.com/OfficeDev/office-js-docs-pr/commits/1b8d2c2716c23b1cb834c526013c861fcd171664)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2049) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2049) |
[docs/outlook/troubleshoot-outlook-add-in-activation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/troubleshoot-outlook-add-in-activation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/troubleshoot-outlook-add-in-activation?branch=pr-en-us-2049) |
[docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/use-regular-expressions-to-show-an-outlook-add-in?branch=pr-en-us-2049) |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview?branch=pr-en-us-2049) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008032118028369-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=RTgAaJ7psCnGp5M4mivdIywuGyxqqu9Lrha1w%2FjJBx4%3D&st=2020-08-03T21%3A14%3A31Z&se=2020-09-03T21%3A19%3A31Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b65e838](https://github.com/OfficeDev/office-js-docs-pr/commits/b65e838e5ef3db6df5b209ea42a6e9e7aa3eafa3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2049) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2049) |
[docs/outlook/troubleshoot-outlook-add-in-activation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/troubleshoot-outlook-add-in-activation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/troubleshoot-outlook-add-in-activation?branch=pr-en-us-2049) |
[docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/use-regular-expressions-to-show-an-outlook-add-in?branch=pr-en-us-2049) |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview?branch=pr-en-us-2049) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008032202502131-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2htWUncbDkNpqoU6lGiHy8V4r3Eoe5%2FUpuzW66fK5A4%3D&st=2020-08-03T21%3A59%3A31Z&se=2020-09-03T22%3A04%3A31Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[595bdbf](https://github.com/OfficeDev/office-js-docs-pr/commits/595bdbf2f1d897cd09b6d5c032bf3de21c7217f9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2049) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2049) |
[docs/outlook/troubleshoot-outlook-add-in-activation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/troubleshoot-outlook-add-in-activation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/troubleshoot-outlook-add-in-activation?branch=pr-en-us-2049) |
[docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/outlook/use-regular-expressions-to-show-an-outlook-add-in.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/use-regular-expressions-to-show-an-outlook-add-in?branch=pr-en-us-2049) |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-irm/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview?branch=pr-en-us-2049) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008032205298433-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=9hs%2BzwExWWmLJYbxSy631C47uU9t0PDWhe%2Fuvw51Kdo%3D&st=2020-08-03T22%3A02%3A07Z&se=2020-09-03T22%3A07%3A07Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**